apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -579,6 +586,8 @@ static RdaLoadOptions readRdaLoadOptionsFromEnvironmentVariables(IdHasher.Config
     readEnvParsedOptional(ENV_VAR_KEY_RDA_JOB_STARTING_MCS_SEQ_NUM, Long::parseLong)
         .ifPresent(jobConfig::startingMcsSeqNum);
     readEnvBooleanOptional(ENV_VAR_KEY_PROCESS_DLQ).ifPresent(jobConfig::processDLQ);
+    jobConfig.rdaVersion(
+        RdaVersion.builder().versionString(readEnvStringRequired(ENV_VAR_KEY_RDA_VERSION)).build());
```

Comment:
We should default this to `"~" + RdaService.RDA_PROTO_VERSION` or something so that build scripts don't break if they fail to include this env var.

Reply:
Now defaulting to using the `RdaSerice.RDA_PROTO_VERSION` defined version, matching MAJOR version (^)